docs/components/featured-content-modules.md

```diff
@@ -116,28 +116,13 @@ variations:
       remains in view at all screen sizes.
     variation_name: Centered visual
 usage: >-
-  <h3>Use cases</h3>
-
-  Use a featured content module when you want to highlight something that’s:
-
-  * Timely: Highlights a new tool or development. Keep in mind: When you remove
-  dated content from a FCM, make sure you're not getting rid of the only way
-  users can navigate to it. Make sure to plan a permanent home for the content.
-
-  * Impactful: Lets people know immediately how the content will affect them,
-  how they can use it, and how it might enhance their understanding of—or
-  experience with—the primary page content
-
-  * Localized: New content that specifically applies to people in a specific
-  geographic area; it may be important for that audience to see their specific
-  content first, and when it’s removed from the page it won’t disrupt the usual
-  user flow.
-
-  * Unique: Content that is specifically in the Bureau’s “wheelhouse” because of
-  our position as an approachable, trusted, unbiased authority
-
-
-  Featured content modules should highlight a single piece of content. That
+  * <h3>Use cases</h3>
```

Comment:
This whole section should be formatted differently. Seems like "Use cases" shouldn't be a bullet and then the Timely, Impactful, etc. bits should be bullets.

<img width="882" alt="Screen Shot 2019-09-26 at 10 54 23 AM" src="https://user-images.githubusercontent.com/704760/65699384-048d0d80-e04c-11e9-9ef4-184fe6a3d183.png">